.github/workflows/ci-kuberenetes-itests.yml

```diff
@@ -0,0 +1,97 @@
+name: Quarkus CI
+
+env:
+  MAVEN_ARGS: -B -e
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    types: [ opened, edited, labeled, reopened, synchronize ]
```

Comment:
a bit concerned abuot enabling this kinda run on all prs...wont we run out of resources quite quickly that way ? :)

Reply:
Agree, this is why the workflow is limited only to prs labelled with `area/kubernetes` (see condition below)